gcloud/datastore/helpers.py

```diff
@@ -115,6 +115,37 @@ def _get_meaning(value_pb, is_list=False):
     return meaning
 
 
+def _new_value_pb(entity_pb, name):
+    """Add (by name) a new ``Value`` protobuf to an entity protobuf.
+
+    :type entity_pb: :class:`gcloud.datastore._generated.entity_pb2.Entity`
+    :param entity_pb: An entity protobuf to add a new property to.
+
+    :type name: string
+    :param name: The name of the new property.
+
+    :rtype: :class:`gcloud.datastore._generated.entity_pb2.Value`
+    :returns: The new ``Value`` protobuf that was added to the entity.
+    """
```

Comment:
I'd rather not. This wasn't about functionality of our library, just about making the switch easier. In fact I'd be happy tearing `_new_value_pb` out after the switch.

The old way to add a property is here:

```python
prop_pb = e_pb.property.add()
prop_pb.name = name
value_pb = prop_pb.value
```

while with `v1beta3` it is simply

```python
value_pb = e_pb.properties.get_or_create(name)
```

since `map`s are supported in `v1beta3`.